src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/TournamentQueries.cs

```diff
@@ -76,28 +79,34 @@ public Tournament Execute(TournamentByGroupCriteria criteria)
                              where g.Id == criteria.GroupId
                              select t;
 
-            return tournament.Select(GetTournamentMapping()).FirstOrDefault();
+            return tournament
+                .Select(GetTournamentMapping())
+                .FirstOrDefault();
         }
 
         /// <summary>
         /// Finds Tournament by given criteria
         /// </summary>
         /// <param name="criteria"> The criteria. </param>
         /// <returns> The <see cref="Tournament"/>. </returns>
-        public List<Tournament> Execute(GetAllCriteria criteria)
+        public ICollection<Tournament> Execute(GetAllCriteria criteria)
         {
-            return _unitOfWork.Context.Tournaments.Select(GetTournamentMapping()).ToList();
+            return _unitOfWork.Context.Tournaments
+                .Select(GetTournamentMapping())
+                .ToList();
         }
 
         /// <summary>
         /// Finds List of Tournament by given criteria
         /// </summary>
         /// <param name="criteria"> The criteria. </param>
         /// <returns> The <see cref="Tournament"/>. </returns>
-        public List<Tournament> Execute(OldTournamentsCriteria criteria)
+        public ICollection<Tournament> Execute(OldTournamentsCriteria criteria)
         {
             return _unitOfWork.Context.Tournaments
+#pragma warning disable S1125 // Boolean literals should not be redundant
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)